.yarnrc.yml

```diff
@@ -1,7 +1,9 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
```

Comment:
why?

Reply:
Set by Yarn itself when updated to the latest major version.

Reply:
https://yarnpkg.com/configuration/yarnrc

> The cache is deemed to be relatively safe to be shared by multiple projects, even when multiple Yarn instances run at the same time on different projects.

This does not make me feel more confident about this 🤔 

Maybe we should stick with this, because cache will be stored in `cacheFolder: "./.yarn/cache",`